src/nvim/auevents.lua

```diff
@@ -127,25 +127,37 @@ return {
     'WinScrolled',            -- after a window was scrolled or resized
   },
   aliases = {
-    BufCreate = 'BufAdd',
-    BufRead = 'BufReadPost',
-    BufWrite = 'BufWritePre',
-    FileEncoding = 'EncodingChanged',
+    {
```

Comment:
re https://github.com/neovim/neovim/issues/19918 , `auevents.lua` may be a file that we want to ship in the runtime, so that plugins can discover Nvim event definitions. In that case, I'd like to keep the "map" structure of this file. So callers would need to use e.g. `spairs()` to get ordering guarantees here. @bfredl ok with this?

This doesn't apply to `keysets.lua`, which is an internal impl detail.